src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -283,6 +283,33 @@ namespace rewrite {
     return RewriteResponse(REWRITE_DONE, working);
   }
 
+  RewriteResponse leqId(TNode node, bool isPreRewrite)
+  {
+    Assert(node.getKind() == kind::FLOATINGPOINT_LEQ);
+
+    if (node[0] == node[1])
+    {
+      NodeManager *nm = NodeManager::currentNM();
+      return RewriteResponse(
+          REWRITE_DONE,
```

Comment:
If this rewrite is used in a postrewrite, we should probably return `REWRITE_AGAIN_FULL` because it could be that `(fp.isNaN x)` can be rewritten to something simpler, e.g. if `x` is a BV-to-FP conversion where we know that the bits of the exponent.

Reply:
Fair point.  In postrewrite ```x``` is in canonical form but ```(fp.isNaN x)``` isn't.

Reply:
Fixed.